tests/wpt/metadata/html/dom/interfaces.html.ini

```diff
@@ -8658,6 +8658,12 @@
   [Location interface: window.location must inherit property "reload" with the proper type (2)]
     expected: FAIL
 
+  [Location interface: existence and properties of interface prototype object]
+    expected: FAIL
+
+  [Stringification of window.location]
+    expected: FAIL
```

Comment:
Yeah, I concluded there was something fishy about that. The second is marked an expected failure in gecko, I'll take another look at the first.